includes/Generator/Customer.php

```diff
@@ -141,9 +140,40 @@ public static function generate( $save = true ) {
 			$customer->save();
 		}
 
+		/**
+		 * Action: Customer generator returned a new customer.
+		 *
+		 * @since 1.2.0
+		 *
+		 * @param \WC_Customer $customer
+		 */
+		do_action( 'smoothgenerator_customer_generated', $customer );
+
 		return $customer;
 	}
 
+	/**
+	 * Create multiple customers.
+	 *
+	 * @param int $amount The number of customers to create.
+	 *
+	 * @return int[]|\WP_Error
+	 */
+	public static function batch( $amount ) {
+		$amount = self::validate_batch_amount( $amount );
+		if ( is_wp_error( $amount ) ) {
+			return $amount;
+		}
+
+		$customer_ids = array();
+
+		for ( $i = 1; $i <= $amount; $i ++ ) {
```

Comment:
```suggestion
		for ( $i = 1; $i <= $amount; $i++ ) {
```

If we apply the same coding standards as we use for WC Core, we should clear the space before the incrementor.